packages/flutter/lib/src/material/pickers/date_picker_dialog.dart

```diff
@@ -487,6 +487,7 @@ class _DatePickerDialogState extends State<_DatePickerDialog> {
         ),
       ),
       insetPadding: const EdgeInsets.symmetric(horizontal: 16.0, vertical: 24.0),
+      // TODO(Piinks): remove once border radius migration is complete
```

Comment:
There is also one of these in the new `date_range_picker_dialog.dart` (line 410). I can update it if you don't want to do it with this PR.

Reply:
Oh thanks! I missed that one! I've added it. :)

Reply:
Nope, you didn't miss it, it was landed after this PR was created. Thanks.